content/en/docs/contribute/participating.md

```diff
@@ -93,33 +93,17 @@ in the Kubernetes organization. Follow these steps:
       SIG Docs member.
       {{< /note >}}
 
-2.  Send an email to the [Kubernetes membership request list](mailto:kubernetes-membership@googlegroups.com)
-    and add your two sponsors and any other relevant people to the CC of the
-    email. Use the following template.
-    
-      ```plaintext
-        I have joined kubernetes-dev@googlegroups.com and fulfilled all the
-        prerequisites outlined at
-        https://github.com/kubernetes/community/blob/master/community-membership.md.
-
-        Sponsors:
-        - Github username / email address
-        - Github username / email address
-
-        List of contributions:
-        - PR URL or other link - description or summary
-        - PR URL or other link - description or summary
-        - PR URL or other link - description or summary
-        - PR URL or other link - description or summary
-        - PR URL or other link - description or summary
-
-        Thanks for your consideration,
-        Your Name
-      ```
-
-3.  Wait for your sponsors to reply, and be available to answer any questions
-    that your sponsors or other Kubernetes leadership has, and for the final
-    result of your application.
+2.  Open a Github issue in the `kubernetes/org` repository to request membership.
+    Fill out the template using the guidelines at
+    [Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md).
+
+3.  Let your sponsors know about the Github issue, either by at-mentioning them
```

Comment:
you say either here, but don't provide two options

Reply:
Fixed